funcx_sdk/funcx/serialize/off_process_checker.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import logging
+import socket
+import argparse
+import pickle
+
+
+def server(port=0, host='', debug=False, datasize=102400):
+
+    from funcx.serialize import FuncXSerializer
+    fxs = FuncXSerializer()
```

Comment:
If the default `use_offprocess_checker` is `True`, this should be explicitly set to `False` then